dev/com.ibm.ws.jsonb_fat/fat/src/com/ibm/ws/jsonb/fat/JSONBInAppTest.java

```diff
@@ -45,6 +47,10 @@ public static void setUp() throws Exception {
         FATSuite.jsonbApp(server);
 
         server.startServer();
+
+        if (JakartaEE10Action.isActive()) { //TODO possibly back port this info message to EE9 and EE8
+            assertTrue(!server.findStringsInLogsAndTrace("JSON5000I").isEmpty());
```

Comment:
I see the following in server.xml (which gets swapped to EE10 features) and so would expect to not see the message:
```
      <feature>servlet-4.0</feature>
      <feature>jsonbContainer-1.0</feature>
      <feature>jsonpContainer-1.1</feature>
```

Reply:
It's a bit odd because jsonp is implicit, but we are also saying it is superseded away.  I hope we don't get users contacting us stating that they have the bell enabled, asking why won't the informational warning go away?

Reply:
Yeah, this message will appear whenever a user uses jsonpContainer alongside a jsonb feature.  But I think that is still okay since it is an informational message and not a warning, and should only be produced when trace is enabled.